composer.json

```diff
@@ -20,10 +20,12 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.2.6",
+        "phpunit/phpunit": "^7.2.6 || ^9",
```

Comment:
The minimum php version of this lib is php 7.2, so the minimum vesion of phpunit could be 8, as that still supports php 7.2